en/docs/fragments/saml-app-front-channel-configs.md

```diff
@@ -0,0 +1,8 @@
+1.   Select **Enable Response Signing**.
```

Comment:
We need to replace the fragment from the portal docs right? Can we add that standard TODO here?